src/nomad_material_processing/vapor_deposition.py

```diff
@@ -326,8 +329,8 @@ class VaporDepositionStep(ActivityStep):
         repeats=True,
     )
     substrate = SubSection(
-        section_def=Substrate,  #### in next commit, I will think and refine better this name and/or structure
-        repeats=True,
+        section_def=Substrate,  
+        #repeats=True, ############################### why is it an array? let's make the substrate inside an array or??
```

Comment:
Each substrate has a different position and could have a different temperature and mask. So I would keep it repeating.

Reply:
okay makes sense. I just saw also the substrate_specimen in line 247 as a repeated section so the two of them were too much. I will remove it from line 247 and keep it here. You confirm?